modules/tor.sh

```diff
@@ -84,6 +89,11 @@ function tor {
       echo "OK."
     elif [ "$option" = "add" ]; then
       value="$3"
+      if [ -n "$4" ]; then
```

Comment:
@Kid243 I am confused now ...

Reply:
Oh this is not supposed to be there anymore, I thought I erased it. First I placed it in the wrong spot by mistake and this statement is no longer needed